che-incubator/che-code
# chore: Adapt Che-Code to the latest VS Code upstream changes

### What does this PR do?
Adapt Che-Code to the latest VS Code upstream changes.

### What issues does this PR fix?

https://github.com/eclipse/che/issues/21984

### How to test this PR?
The conflicts are related to these changes https://github.com/che-incubator/che-code/pull/163
So, please see the corresponding PR ^
